### DISCUSSION TITLE
Workflow status images in Readme/repository view for private repositories don't work

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44627] [Mobile] [answered]
### Select Topic Area

Bug

### Body

GitHub Mobile beta

Browse to a repository with a readme with a workflow status image.

If it's a public repository, like:
https://github.com/check-spelling/check-spelling 

![Screenshot_20230117-191127](https://user-images.githubusercontent.com/2119212/213047663-75efdc97-ebe9-4e66-8e03-a7766897765f.png)

If it's a private repository:
![Screenshot_20230117-191151](https://user-images.githubusercontent.com/2119212/213047778-0f8688f1-c473-4020-9383-a99123153a33.png)
That same repository from browser:
![Screenshot_20230117-191046~2](https://user-images.githubusercontent.com/2119212/213047914-dc22826f-1d75-49e4-8aba-2e01e084702b.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref we are aware of this. Adding this report to the list. Thanks!

**Comment by bxbdbdbj** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> GitHub Mobile beta
> 
> 
> 
> Browse to a repository with a readme with a workflow status image.
> 
> 
> 
> If it's a public repository, like:
> 
> https://github.com/check-spelling/check-spelling 
> 
> 
> 
> ![Screenshot_20230117-191127](https://user-images.githubusercontent.com/2119212/213047663-75efdc97-ebe9-4e66-8e03-a7766897765f.png)
> 
> 
> 
> If it's a private repository:
> 
> ![Screenshot_20230117-191151](https://user-images.githubusercontent.com/2119212/213047778-0f8688f1-c473-4020-9383-a99123153a33.png)
> 
> That same repository from browser:
> 
> ![Screenshot_20230117-191046~2](https://user-images.githubusercontent.com/2119212/213047914-dc22826f-1d75-49e4-8aba-2e01e084702b.png)
> 
> 
> 
>

**Comment by andrian-vangelov** [comment]
The problem still persists.

It has been two years now, do you have any news on that?

**Reply by stevepopovich** [reply]
At this time, this is still in our backlog.
